### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by simonschmeisser** [ompl/ompl#868]
Some important bugs have been fixed and it would be nice to have a new (patch) release. Both for ros distribution as well as for getting those bug fixes included with Ubuntu 22.04

@mamoll @zkingston 

E.g.: https://github.com/ros-planning/moveit/issues/1920 via #739

**Comment by mamoll** [comment]
We're planning to do this soon. @gammell are there any last-minute things you'd want to include in a patch-level release?

**Comment by gammell** [comment]
I don't think we are sitting on any patch-level changes at the moment. @marlinstrub would know best if we have anything, but he's very busy at the moment, so you shouldn't necessarily wait on his response.

**Comment by k2shah** [comment]
bump!
any chance of a new release?

**Comment by simonschmeisser** [comment]
Friendly ping

**Comment by mamoll** [comment]
@zkingston?

**Comment by zkingston** [comment]
Added to the todo list - hopefully will push out within a week or so.

**Comment by zkingston** [comment]
1.6.0 is out now!
